README.md

```diff
@@ -17,7 +17,7 @@ instance roles)
 
 # Cloud Providers
 
-Currently the operator supports AWS, GCP, Azure, VMWare, OpenStack and oVirt.
+Currently the operator supports AWS, Azure, GCP, kubeVirt, OpenStack. oVirt and VMWare.
```

Comment:
The constant in the Infrastructure API looks like "KubeVirt" which would look a little more natural to me here, could we sync up on that casing across the docs/PR for consistency?

Reply:
I fixed all the cases in documentation